saleor/discount/migrations/0084_set_missing_currency.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 4.2.16 on 2025-01-14 14:28
+
+from django.db import migrations
+from django.db.models import OuterRef, Subquery
+
+# The batch of size 5000 takes ~0.1 second and consumes ~4MB memory at peak
+BATCH_SIZE = 5000
+
+
+def set_discount_currency(apps, _schema_editor):
+    OrderDiscount = apps.get_model("discount", "OrderDiscount")
+    qs = (
+        OrderDiscount.objects.filter(currency="")
+        .select_related("order")
+        .order_by("id")
+        .only("id", "currency", "order__currency")
```

Comment:
This statement is redundant, as we will do `values_list` on that `qs` anyway in `queryset_in_batches`